crates/ironrdp-web/src/session.rs

```diff
@@ -1039,8 +1077,8 @@ where
                 } => (x224_connection_response, server_cert_chain, server_addr),
                 ironrdp_rdcleanpath::RDCleanPath::Err(error) => {
                     return Err(
-                        IronRdpError::from(anyhow::anyhow!("received an RDCleanPath error: {error}"))
-                            .with_kind(IronRdpErrorKind::RDCleanPath),
+                        RemoteDesktopError::from(anyhow::anyhow!("received an RDCleanPath error: {error}"))
+                            .with_kind(RemoteDesktopErrorKind::RDCleanPath),
```

Comment:
note: Again, not your code, but it’s better to do this:

```suggestion
                        RemoteDesktopError::from(anyhow::Error::new(error).context("received an RDCleanPath error"))
                            .with_kind(RemoteDesktopErrorKind::RDCleanPath),
```

Reply:
Done